code/modules/clothing/under/miscellaneous.dm

```diff
@@ -37,9 +37,9 @@
 	species_fit = list(GREY_SHAPED)
 
 /obj/item/clothing/under/sl_suit/armored
-	desc = "It's a very amish looking suit. This one looks armored."
+	desc = "It's a very amish looking suit. This one has a tag. It says "Gorlex" in cursive."
```

Comment:
It says "Gorlex" in cursive."

Change it to It says It says \"Gorlex\" in cursive."

Reply:
Alright, will do. Any idea why the check is failing, exactly?

Reply:
I guess that was it. Thanks. Can't put use -"- in code frivolously, should have realised.